protected/humhub/modules/space/models/Space.php

```diff
@@ -261,6 +261,12 @@ public function afterSave($insert, $changedAttributes)
             $activity->create();
         }
 
+        if ($this->visibility != $changedAttributes['visibility']) {
+            if ($this->visibility == Space::VISIBILITY_NONE) {
+                Content::updateAll(['visibility' => Content::VISIBILITY_PRIVATE], ['contentcontainer_id' => $this->contentcontainer_id]);
```

Comment:
In https://github.com/humhub/humhub/issues/4713 we've discussed about maybe using an ActiveJob for this in order to be able to handle content followers etc, which may not be able to see the content anymore. I think we did not come to a final decision yet or did you think about another solution for this?